### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by fermadeiral**

### CODE REVIEW

File: doc/usage/repairnator-config.md [hunk 1]
```diff
@@ -383,7 +383,7 @@ It defines the name of the docker image to use for Repairnator BEARS checkbranch
 
 Its default value is `spirals/checkbranches:latest`.
 
-### Root pathes
+### Root paths
```
Comment (surli) [inline doc/usage/repairnator-config.md]: One day I'll know about that one, one day, maybe...

Comment (fermadeiral) [reply doc/usage/repairnator-config.md]: hahaha

**Comment by fermadeiral** [comment]
All the builds based on the current master will fail until #715 be merged, because the schemas in the docker-images folder are the old ones, which do not match the new testing branches.

**Comment by surli** [comment]
Yes I'm aware of that. Do you plan to fix other typo?

**Comment by fermadeiral** [comment]
I do, but I have no guarantee. I think you can merge this one when you want. I propose a new one later if I do more changes.

**Pull request merged**